www/obtaining.rst

```diff
@@ -0,0 +1,58 @@
+.. _obtaining:
+
+===============
+Obtaining NumPy
+===============
```

Comment:
I'm not convinced about adding this page. It is in general not the right advice for beginning users, which are the main set of users that will follow the `obtaining` link probably. I'd much rather just link to https://scipy.org/install.html.

Reply:
This is meant to eventually replace that outdated information with our current "best practices". What is not the right advice?

Reply:
ok, backing this out